### ISSUE TITLE
Support 64-bit

### ISSUE THREAD

**Issue by ProgrammerWhoPrograms** [AgregoreWeb/agregore-mobile#36]
![Warning that Agregore is 32-bit and GrapheneOS will disable 32-bit apps for security](https://github.com/user-attachments/assets/6b8223de-0f65-457c-a651-db606d91cabd)
https://discuss.grapheneos.org/d/1968-32bit-apps-on-google-pixel-7

**Comment by RangerMauve** [comment]
Dang, yeah right now we're limited in that we don't have a build server anymore. If folks could donate a VM or compiling the app I could take a shot at adding it.

**Comment by ProgrammerWhoPrograms** [comment]
Would some money help?

**Comment by RangerMauve** [comment]
Probably yes, but I'm aiming for at least 60k USD so we can get a decent chunk of work done at once.

**Comment by ProgrammerWhoPrograms** [comment]
I can't afford that much lol

**Comment by RangerMauve** [comment]
Yeah I don't expect individuals to do so. If you come across any grant funds then it'd be more likely. I'm on the lookout as well./ In the meantime if you have a beefy computer you couuld try cloning the repo and doing a build yourself. You may need to leave it running overnight

**Comment by RangerMauve** [comment]
You'd also need to compile a 64bit version of this repo: https://github.com/AgregoreWeb/agregore-ipfs-daemon
